openfe/setup/atom_mapping/ligandatommapping.py

```diff
@@ -2,12 +2,15 @@
 # For details, see https://github.com/OpenFreeEnergy/openfe
 import json
 import gufe
-from typing import Any, Optional
 import numpy as np
+from typing import Any, Optional
 from numpy.typing import NDArray
 
+from py3Dmol import view
```

Comment:
We'll probably want to make this an optional dependency like perses. py3dmol can be quite heavy with all the node.js things